src/transformers/models/conditional_detr/image_processing_conditional_detr.py

```diff
@@ -1281,7 +1281,7 @@ def preprocess(
         return encoded_inputs
 
     # POSTPROCESSING METHODS - TODO: add support for other frameworks
-    def post_process(self, outputs, target_sizes):
+    def post_process(self, outputs, target_sizes, top_k: int = 300):
```

Comment:
This method is deprecated in favor of `post_process_object_detection`, cc @amyeroberts

Reply:
I could remove the changes from the deprecated post_process functions if preferred?

Reply:
Sure, I reverted the changes to the deprecated methods.